src/ui/atoms/fileInput/FileInput.tsx

```diff
@@ -0,0 +1,183 @@
+import React, { useCallback } from 'react'
+import './fileInput.scss'
+import classNames from 'classnames'
+import fileInputIcon from '../../../assets/images/file.png'
+import { Typography } from '../typography/Typography'
+import type { DeepReadonly } from 'superTypes'
+
+export type FileUploadProps = {
+  /**
+   * The unique id of the component
+   */
+  readonly id: string
```

Comment:
Sorry to come back to this point, but it bothers me to expose an identifier only used to allow the implementation of the component, when it can be done internally. 

From what I see, using a Ref should be fine for the what we want to achieve.

Reply:
I just had an idea on that point using the `classNames`. Maybe we do not need this prop anymore and even the `getInputFileElement` function you mentioned above.
I will work on it and submit the PR.